src/nft/components/collection/CollectionAssetLoading.tsx

```diff
@@ -0,0 +1,28 @@
+import { loadingAsset } from 'nft/css/loading.css'
+
+import SizingImage from '../../../assets/images/sizing-image.png'
+import { Box } from '../../components/Box'
+import { Row } from '../Flex'
+import * as styles from './CollectionAssetLoading.css'
+
+export const CollectionAssetLoading = () => {
+  return (
+    <Box as="div" className={styles.collectionAssetLoading}>
+      <Box as="div" position="relative" width="full">
+        <Box as="div" className={styles.collectionAssetsImageLoading} />
+        <Box as={'img'} width="full" opacity="0" src={SizingImage} draggable={false} />
```

Comment:
id the loading image the only way to achieve this effect? Could we set a fixed height and width and set flexShrink to 0? I think we should avoid hardcoding in a random nft to use for sizing

Reply:
The height and width change proportionally to screen size.  Fixed size will not look as good in my opinion.  The alternative to this is making the asset cards height not dependent on the image, which I would prefer not to do.

Reply:
I sent a blank 250x250 png to use, also the brackets around `img` can be removed